src/components/PositionedOverlay/utilities/math.ts

```diff
@@ -91,6 +91,7 @@ export function calculateHorizontalPosition(
   preferredAlignment: PreferredAlignment,
 ) {
   const maximum = containerRect.width - overlayRect.width;
+  const borderWidth = 2;
```

Comment:
Whats this borderWidth for?

Reply:
Additional width on the left and right of the popover. It wasnt accommodated before unfortunately.